outbound-dialer/src/main/java/org/motechproject/nms/outbounddialer/service/impl/TargetFileServiceImpl.java

```diff
@@ -130,6 +136,16 @@ private String targetFileName() {
     }
 
 
+    private void insertTargetFileAuditRecord(TargetFileNotification tfn, String status) {
+        if (tfn == null) {
+            fileAuditDataService.create(new AuditRecord(FileType.TargetFile, null, null, null, status));
+        } else {
+            fileAuditDataService.create(new AuditRecord(FileType.TargetFile, tfn.getFileName(), tfn.getRecordCount(),
+                    tfn.getChecksum(), status));
+        }
+    }
+
+
     private File createTargetFileDirectory() {
         File userHome = new File(System.getProperty("user.home"));
         File targetFileDirectory = new File(userHome, settingsFacade.getProperty(TARGET_FILE_DIRECTORY));
```

Comment:
a network share yes, but not an ftp server.  Also I worry about maintaining that mount across all servers in the cluster.

I don't have the SRS open, but we do need to know if it is a network share or ftp.  (I don't know, you might know better than me).  If not just mark it with a todo and a ticket.